### ISSUE TITLE
Linting diagnostics for valid tripe-slash directives

### ISSUE THREAD

**Issue by kitsonk** [denoland/deno_lint#718]
### Lint Name

no-invalid-triple-slash-reference

### Code Snippet

```ts
/// <reference no-default-lib="true" />
```

### Expected Result

No diagnostics.

### Actual Result

```
Invalid format of triple-slash reference directive
Correct format is `/// <reference xxx="some_value" />` where `xxx` is one of `types`, `path`, and `lib`
```

### Additional Info

We should allow all proper triple-slash references in a TypeScript file.

### Version

Deno Canary

**Comment by magurotuna** [comment]
Thanks, I forgot to let the allow list include `no-default-lib`. I'll fix it.
